(AzureCXP) fixes MicrosoftDocs/azure-docs#59903 (#72704)
Azure/azure-content: articles/active-directory-b2c/add-password-change-policy.md

1. Open the policy that you changed. For example, *B2C_1A_profile_edit_password_change*.
2. For **Application**, select your application that you previously registered. To see the token, the **Reply URL** should show `https://jwt.ms`.

> 3. Click **Run now**. A URL will be opened in a new tab, remove "&prompt=login" from the end and refresh. Sign in with the account that you previously created. You will now have the opportunity to change the password.

Reviewer: ```suggestion
3. Click **Run now**. In the new tab that opens, remove "&prompt=login" from the URL and refresh the tab. Then sign in with the account you previously created. You will now have the opportunity to change the password.
```
